### DISCUSSION TITLE
Lock function PLEASE

### DISCUSSION THREAD

**Discussion by Halproboardstest** [zen-browser/desktop#4003] [Ideas]
I'm  loving ZEN, but would request one feature, when you go to edit a workspace or even tabs, could you add a lock function, so as to make it harder for idiots like me, from deleting things they actually meant to keep :)

So a simple Lock / Unlock function - I.e. no more tabs can be added or more to the point in my case DELETED from this workspace, until this workspace is unlocked, also it would be useful if it could be applied to tabs. I accidentally deleted all my tabs in one of my work spaces and I was like oooops, how do I get them back... maybe an undo last action button as well.... thanks in advance and keep up the amazing work.

With much respect and a bit of hope.

**Comment by DRYN07** [comment]
I would love it. I'm working on a bunch of projects, each with its own sideberry panel. If I just click once to "delete all tabs below," I lose everything.......... xd
